themes/theme-ios11/components/AppBar/presets/DefaultBar/connector.js

```diff
@@ -17,8 +17,8 @@ function makeMapStateToProps() {
  * @returns {Object}
  */
 const mapDispatchToProps = dispatch => ({
-  updateStatusBar: ({ background }) => {
-    dispatch(updateStatusBarBackground(background));
+  updateStatusBar: ({ background }, setDefault) => {
```

Comment:
Same as before. The name might introduce confusion about what `setDefault` actually is.